src/SFA.DAS.Employer.PR.Web.UnitTests/SFA.DAS.Employer.PR.Web.UnitTests.csproj

```diff
@@ -9,6 +9,7 @@
   <ItemGroup>
     <PackageReference Include="FluentAssertions" Version="7.0.0-alpha.4" />
     <PackageReference Include="NUnit" Version="4.1.0" />
+    <PackageReference Include="SFA.DAS.Encoding" Version="1.1.76" />
```

Comment:
You do not need to explicitly reference packages here if the referenced project has already got it. The package comes as indirect dependency.

Reply:
done